src/DataGrid.tsx

```diff
@@ -118,8 +119,10 @@ export interface DataGridProps<R, SR = unknown> extends SharedDivProps {
   sortColumn?: string;
   /** The direction to sort the sortColumn*/
   sortDirection?: SortDirection;
+  /**Used for multi column sorting */
+  sortData?: Sorts[];
   /** Function called whenever grid is sorted*/
-  onSort?: (columnKey: string, direction: SortDirection) => void;
+  onSort?: (columnKey: string, direction: SortDirection, ctrlClick: boolean) => void;
```

Comment:
I think we can replace these 4 props with 2 new props

```
sortColumns?: readonly Readonly<SortColumn>[];
onSortColumnsChange?: (sortColumns: SortColumn[]) => void;
```

If the sortColumns.length === 1 then it is a single sort 

```
interface SortColumn {
  columnKey: string;
  direction: SortDirection;
}
````